packages/ovm/test/bridge/L1ToL2TransactionPasser.spec.ts

```diff
@@ -1,29 +0,0 @@
-import '../setup'
-
-/* External Imports */
-import { L1ToL2TransactionPasserContractDefinition } from '@eth-optimism/rollup-contracts'
-
-import { createMockProvider, deployContract, getWallets } from 'ethereum-waffle'
-
-/* Contract Imports */
-describe('L1 -> L2 Transaction Passer', () => {
-  let provider
-  let wallet
-  let l1ToL2TransactionPasser
-  beforeEach(async () => {
-    provider = createMockProvider()
-    wallet = getWallets(provider)[0]
-    l1ToL2TransactionPasser = await deployContract(
-      wallet,
-      L1ToL2TransactionPasserContractDefinition
-    )
-  })
-  const entryPoint = '0x0000000000000000000000000000000000001234'
-  const callData = '0xdeadbeefee5555'
-  it('should emit the correct L1ToL2Transaction L1 event when an L1->L2 tx is sent', async () => {
-    await l1ToL2TransactionPasser
```

Comment:
Are these tests reflected anywhere in these changes or just generally deleted? Do these contracts impose a circular dependencies? If they do maybe putting stuff in `rollup-full-node` makes sense? Although ideally these tests are just in the `rollup-contracts` package

Reply:
These tests have all been transferred over to the `rollup-contracts` pacakge.